tests/test_notifications.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from foxpuppet.browser.notifications import (
+    BaseNotification,
+    AddOnInstallBlocked)
+
+
+@pytest.fixture
+def firefox_options(firefox_options):
+    firefox_options.add_argument('-foreground')
```

Comment:
OK, thanks. I think I’ve asked about this before, but what is a ‘popup notification’ in this case?